plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/general/Description.tsx

```diff
@@ -70,3 +70,9 @@ const SwitchWrapper = styled.div`
   gap: 10px;
   margin-top: 12px;
 `;
+
+const Markdown = styled.div`
+  img {
+    width: 100%;
```

Comment:
Is it possible to do max-width: 100% and still successfully fix this issue? 
The problem with setting width to 100% is that ALL images, all the time will be 100%, even if the image is smaller than the width of the DatasetCard

Reply:
Makes sense! Done!